docs: Improve documentation for Unit and Option (#11)
taxfyle/FxKit: docs/core/unit.md

    // Alternatively:
    // return default;
}
```

## Prelude

The `Prelude` class provides the following functions for `Unit`:

### Unit Function

`Unit` value can be returned using the `Unit()` function.

```csharp
public Result<Unit, string> DoSomething()
{
    // Do something
    return Unit();
}
```

### Ignore Function

> The `Ignore` function allows to transform any value to a `Unit` value.

Reviewer: ```suggestion
The `Ignore` function transforms any value into a `Unit` value.
```

Author: Done
